syntax = "proto3";

package swarm;

import "simulation/swarm/proto/state.proto";
import "simulation/swarm/proto/physical_config.proto";

// Missile configuration.
message MissileConfig {
  // Initial state.
  State initial_state = 1;

  // Physical configuration.
  PhysicalConfig physical_config = 2;
}
